src/pages/dev/general-message-passing/gas-services/pay-gas.mdx

```diff
@@ -50,92 +50,47 @@ contract SimpleTransferContract {
 }
 ```
 
-The `msg.value` is the gas amount we pay to the `AxelarGasService` contract.
-
-So, on the front-end side, we need to pass `sourceGasFee` to `msg.value` like below:
+`msg.value` is the gas amount paid to the `AxelarGasService` contract. Thus, `sourceGasFee` must be passed to `msg.value` on the front end:
```

Comment:
I want to clarify that you aren't passing a variable in, you need to pass the VALUE you got from `sourceGasFee`. Any ideas?

Reply:
I removed references to `sourceGasFee` entirely (except in a snippet where it's clearly marked as a variable that stores the return value of `estimateGasFee()`) and replaced it with references to call `estimateGasFee()`.